package.json

```diff
@@ -66,8 +66,9 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "prebuild": "npm run resource-descriptors",
+    "prebuild": "npm run resource-descriptors && npm run robots",
```

Comment:
Can you not have two items on the line? `npm run resource-descriptors robots`

Reply:
Hmm, I'm not sure. I can try it though.

Reply:
Plus this can be in the post build doesn't have to be in the prebuild.

Reply:
The double ampersand appears to be necessary unfortunately .